src/components/v2/ValueUpdate/index.tsx

```diff
@@ -2,39 +2,57 @@
 import React from 'react';
 import Typography from '@mui/material/Typography';
 import BigNumber from 'bignumber.js';
-import { getBigNumber, formatCentsToReadableValue } from 'utilities/common';
+import { formatCentsToReadableValue } from 'utilities/common';
 import { Icon } from '../Icon';
 import { useStyles } from './styles';
 
-export interface IValueUpdateProps {
+type NumberValueProps = {
+  original: number | undefined;
+  update: number | undefined;
+  format?: (value: number | undefined) => string;
+};
+
+type BigNumberValueProps = {
+  original: BigNumber | undefined;
+  update: BigNumber | undefined;
+  format?: (value: BigNumber | undefined) => string;
+};
+
+export type ValueUpdateProps = {
```

Comment:
nit: This could be an interface and then the type would be the union and intersection of values

Reply:
Sure, works for me